### DISCUSSION TITLE
Open ethereum wallet, different from MetaMask, in iOS app via wallet-connect

### DISCUSSION THREAD

**Discussion by 0xHristo** [WalletConnect/walletconnect-monorepo#1132] [Developer Support]
Currantly I am following the [Wallet Connect's documentation for mobile deeplinking][1].

The deeplink which I am opening is:
```
wc:a593bc04-32b5-4e25-aaec-9e2c247e868a@1?bridge=https%3A%2F%2Fv.bridge.walletconnect.org&key=be7212e692f047b39425b0fe88d805582ac29aa76ccb893d5c0cdef8845dc3c3
```

The problem with the above link is that it always opens the MetaMask mobile app despite having Coinbase wallet and Crypto.com Defi wallet.

What to change in the above deeplink in order to open wallet different from MetaMask?

  [1]: https://docs.walletconnect.com/mobile-linking

**Comment by hackyguru** [comment]
Thanks for taking time to create a discussion thread. This seems to be a problem with your application's preferences (the user could have pressed 'always open with MetaMask' during the initial run. I would recommend you to try clearing the app's data / uninstalling the app and then trying again.

Another possible problem might be the wallet list. Make sure you've included all the wallets that your dApp needs to support. You can learn more at - https://docs.walletconnect.com/mobile-linking#wallet-support and https://docs.walletconnect.com/quick-start/dapps/react-native#walletconnectprovider
